doc/api/cli.md

```diff
@@ -371,12 +371,10 @@ added: v11.12.0
 
 Enable experimental frozen intrinsics like `Array` and `Object`.
 
-Support is currently only provided for the root context and no guarantees are
-currently provided that `global.Array` is indeed the default intrinsic
-reference. Code may break under this flag.
+Only the root context is supported. There is no guarantee that `global.Array`
+is indeed the default intrinsic reference. Code may break under this flag.
```

Comment:
nit: for built-in ES features, I think it makes more sense to use `globalThis` (ES built-in as well) to refer to the global object.
```suggestion
Only the root context is supported. There is no guarantee that
`globalThis.Array` is indeed the default intrinsic reference. Code may break
under this flag.
```